Zotero/Scenes/Master/MasterContainerViewController.swift

```diff
@@ -304,34 +316,38 @@ final class MasterContainerViewController: UINavigationController {
             }
         }
     }
-
+    
     override func viewIsAppearing(_ animated: Bool) {
         super.viewIsAppearing(animated)
-
+        
         updateBottomPosition()
         if let splitViewController {
             // Split view controller collapsed status when the app launches is correct here, so it's used to show/hide bottom sheet for the first appearance.
             // The app may be launched in collapsed mode, if it was in such mode the last time it was moved to background.
             setBottomSheet(hidden: splitViewController.isCollapsed)
         }
     }
-
+    
     override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         super.viewWillTransition(to: size, with: coordinator)
-
-        let visibleHeight = size.height - keyboardHeight
-        set(bottomPosition: bottomPosition, containerHeight: visibleHeight)
-
+        
+        set(bottomPosition: bottomPosition, containerHeight: size.height, keyboardHeight: keyboardHeight)
+        
         coordinator.animate(alongsideTransition: { _ in
             self.view.layoutIfNeeded()
         }, completion: nil)
+        coordinator.animate { _ in
```

Comment:
Why does the coordinator animate view layout changes twice? Wasn't there supposed to be just `completion` block added to existing `coordinator.animate(alongsideTransition: ...`?

Reply:
Didn't delete the old code 😅 Good catch. Check https://github.com/zotero/zotero-ios/pull/791/commits/22bf5330bce3f553ed23f6183aca53263ccca839